lib/datadog/profiling/http_transport.rb

```diff
@@ -60,10 +61,12 @@ def export(flush)
               "Failed to report profiling data (#{config_without_api_key}): " \
               "server returned unexpected HTTP #{result} status code"
             )
+            Datadog::Core::Telemetry::Logging.error("Failed to report profiling data: unexpected HTTP #{result} status code")
             false
           end
         else
           Datadog.logger.error("Failed to report profiling data (#{config_without_api_key}): #{result}")
+          Datadog::Core::Telemetry::Logging.error("Failed to report profiling data")
           false
         end
```

Comment:
I am aware of the verbosity of those curated description. Currently it work around the several constraints/risk for using telemetry logs:

1. Must not transfer PII
2. Reduce the cardinality in order to hash and batch 

Combining the logger and telemetry logs in a single API is risky. Perhaps we can revisit the design in the future with more patterns and concrete examples.

Reply:
Ah. I wasn't aware of the requirement to reduce cardinality. I see how omitting details in these strings will have that effect.

Is this perhaps worth documenting in the telemetry logging class? E.g. So that if/when this question comes up again, other members of the team will have somewhere to reference.

Reply:
Will do that in the followup PR.